README-PNL.md

```diff
@@ -303,11 +305,18 @@ and
 
 ### Cookies
 
-Within DPdash, [cookies are not transferred](http://docs.neuroinfo.org/dpdash/en/latest/building.html?highlight=cookies) over unencrypted HTTP. 
-For this limitation, after entering admin credentials in DPdash login page, you may be dropped back to login again. 
-To circumvent this issue, set the following to `false`:
+By default, DPdash [cookies are not transferred](http://docs.neuroinfo.org/dpdash/en/latest/building.html?highlight=cookies) over unencrypted HTTP. 
+
+Due to this limitation, after entering admin credentials in the DPdash login page, you may be dropped back to login again. This will occur when you are using the app over HTTP or do not have a valid HTTPS certificate for your domain. To circumvent this issue, set the following to `false`:
 
     ${state}/dpdash/configs/dashboard/config.js: config.session.cookie.secure = false;
 
 and then [restart](#restart) the DPdash application.
 
+Note that this is not secure and should only be used in a development environment. A production environment should be served over HTTPS with a valid certificate, and with this set to `true`.
+
+See the [express-session docs](https://expressjs.com/en/resources/middleware/session.html) for more information.
+
+**Note**: The developers use a reverse proxy to serve the production DPdash instance, and thus we have set the following to `true` by default. If you do not use a reverse proxy and want stricter security settings, you may set it to `false` or remove it altogether:
```

Comment:
Is `false` and no proxy equivalent to `stricter security settings`? Should it be non strict?

Reply:
This is referring to the combination of `cookie.secure = true` and `proxy = false`, which is the strictest possible.
`cookie.secure = true` and `proxy = false` is our default config, slightly less strict.
`cookie.secure = false` is always the least strict.